### ISSUE TITLE
Computed Object property generates broken syntax

### ISSUE THREAD

**Issue by miljan-aleksic** [Rich-Harris/buble#92]
Hi and thanks for such great products @Rich-Harris,

there is a small issue depending on how the computed properties are ordered:

```js
// this is well transpiled
const ref = {
  'foo': bar,
  [`foo-${bar}`]: fooBar
}

// becomes
var ref = {
  'foo': bar
};
ref[("foo-" + bar)] = fooBar
```

```js
// this is is not
const ref = {
  [`foo-${bar}`]: fooBar,
  'foo': bar
}

// becomes
var ref = {};
ref[("foo-" + bar)] = fooBar;
ref.'foo' = bar // where this is unvalid JS syntax
```

As temporal workaround is to keep the computed properties always last, but it would be nice if this could be solved. I don't have much experience with this project, sorry for not having any PR instead.

**Comment by adrianheine** [comment]
#91 might be a fix for this.

**Comment by adrianheine** [comment]
Fixed with #91.
